tests/enterprise-contract/contract.go

```diff
@@ -18,26 +27,51 @@ import (
 var _ = framework.EnterpriseContractSuiteDescribe("Enterprise Contract E2E tests", Label("ec", "HACBS"), func() {
 
 	defer GinkgoRecover()
+	var kubeClient *framework.ControllerHub
 	var fwk *framework.Framework
+
 	var err error
 	var namespace string
 	var defaultECP *ecp.EnterpriseContractPolicy
 	var imageWithDigest string
 	var pipelineRunTimeout int
-	var generator tekton.VerifyEnterpriseContract
 	var verifyECTaskBundle string
 	publicSecretName := "cosign-public-key"
+	var generator = tekton.VerifyEnterpriseContract{
```

Comment:
So you probably don't want to do this here. This means that the same instance of the generator will be used for each test. This is very much likely going to cause timing issues and unexpected behavior. (Test X affects the outcome of Test Y.)

I suggest declaring the variable here as it was before, and only instantiate it in a `BeforeEach` block.